types/web-bluetooth/index.d.ts

```diff
@@ -61,12 +62,14 @@ interface BluetoothLEScan extends RequestLEScanOptions {
 }
 
 interface BluetoothAdvertisementEvent extends Event {
```

Comment:
This is called `BluetoothAdvertisingEvent`

```suggestion
interface BluetoothAdvertisingEvent extends Event {
```

Reply:
As this PR introduces breaking changes, suggest this naming is also fixed

Reply:
Agree, was considering it, we may as well align naming here too. I'll get it done.